meson.build

```diff
@@ -192,7 +192,7 @@ install_data(
     install_dir: udev_dir / 'rules.d')
 
 package_datadir = join_paths('share', 'vpd')
```

Comment:
I think you are missing get_option('datadir'). Please check

Reply:
should we need it?
As this is the path I see in logs,
Installing subdir /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/50003000.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/50003000_v2.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/vpd_inventory.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/50001002.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
:Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/50001001_v2.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/50001000.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
:Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/50001001.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
1.0+gitAUTOINC+8bd9124c54-r1/temp/log.do_install.2718989:15:Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/systems.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd
1.0+gitAUTOINC+8bd9124c54-r1/temp/log.do_install.2718989:16:Installing /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/git/config/ibm/50001000_v2.json to /home/girik/GHE/openbmc/build/p10bmc/tmp/work/armv7ahf-vfpv4d16-openbmc-linux-gnueabi/ibm-vpd-parser/1.0+gitAUTOINC+8bd9124c54-r1/image/usr/share/vpd

Reply:
right.. This package_datadir = join_paths('share', 'vpd') is working to install in the right directory.
Just a question here, Do you know from where it is taking /usr as prefix?

Reply:
It's from openbmc recipe, it places to the right path there.